Gemfile

```diff
@@ -33,6 +33,8 @@ gem 'bootsnap', '>= 1.4.4', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'dotenv-rails'
```

Comment:
I don't think you need this gem.
Can you please try without it.

Reply:
In this case the gem is needed, I tried without the gem and the API request didn't work